launcher/src/components/UI/node-manage/ChangeConfirm.vue

```diff
@@ -22,7 +22,7 @@
           <span>{{ item.content }}</span>
           <div class="right-icon">
             <img
-              src="../../../../public/img/icon/manage-node-icons/plugin-item-icon.png"
+              :src="item.service.sIcon"
```

Comment:
change it to item.service.icon

Reply:
ok,
I just thought that they look very small and are very similar as well so it might be hard do differentiate.